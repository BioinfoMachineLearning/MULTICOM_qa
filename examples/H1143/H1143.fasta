>H1143 CNPase-Nb10e, mouse/alpaca, subunit 1, 219 residues;
GLEKDFLPLYFGWFLTKKSSETLRKAGQVFLEELGNHKAFKKELRHFISGDEPKEKLELVSYFGKRPPGVLHCTTKFCDYKAAGAEEYAQQEVVKRSYGKAFKLSISALFVTPKTAGAQVVLTDQELQLWPSDLDKPSASEGLPPGSRAHVTLGCAADVQPVQTGLDLLDILQQVKGGSQGEAVGELPRGKLYSLGKGRWMLSLTKKMEVKAIFTGYYG
>H1143 CNPase-Nb10e, mouse/alpaca, subunit 2, 131 residues;
EVQLEESGGGLVQPGGSLRLSCAVSGITLDDFGIGWFRQAPGKEREGVACISPGYEHIYYADSAKGRFTISRDNAKNTVYLQMNNLKPWDTGVYYCAADNDLPDRLWGGSDWSDPSPYDYWGQGTQVTVSS