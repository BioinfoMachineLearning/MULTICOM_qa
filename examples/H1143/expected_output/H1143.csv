Name,ICPS_01,average_ICPS,average_MMS,Final_score,ICPS_Rank,MM_Rank,Final_Rank
H1143TS035_3,0.1474294117647059,0.1474294117647059,0.6747099999999999,0.4637977647058823,1,2,1.6
H1143TS035_4,0.07450555555555555,0.07450555555555555,0.654876,0.4227278222222222,3,3,3.0
H1143TS035_5,0.13665217391304346,0.13665217391304346,0.6851980000000001,0.46577966956521744,2,1,1.4
H1143TS037_3,0.0016650000000000005,0.0016650000000000005,0.6083999999999999,0.365706,6,5,5.4
H1143TS399_4,0.04980416666666667,0.04980416666666667,0.653512,0.41202886666666666,4,4,4.0
H1143TS493_2,0.005350000000000001,0.005350000000000001,0.602128,0.3634168,5,6,5.6
